MatrixSDK/Utils/MXHTTPClient.m

```diff
@@ -204,6 +204,11 @@ - (void)tryRequest:(MXHTTPOperation*)mxHTTPOperation
         path = [path stringByAppendingString:[NSString stringWithFormat:@"%@access_token=%@", urlSeparator, accessToken]];
     }
     
+    // Remove the potential leading slash for the provided path, so that NSURL +URLWithString:relativeToURL: works as expected.
```

Comment:
Then I think we should just follow the Victor's suggestion: "Make kMXContentPrefixPath changeable or properties"